blktap3_stats_stubs.c

```diff
@@ -28,6 +28,22 @@
 #include <caml/fail.h>
 #include <caml/unixsupport.h>
 
+#define BT3_LOW_MEMORY_MODE 0x0000000000000001
+
+struct stats {
+	uint32_t version;
+	unsigned long long oo_reqs;
+	unsigned long long read_reqs_submitted;
+	unsigned long long read_reqs_completed;
+	unsigned long long read_sectors;
+	unsigned long long read_total_ticks;
+	unsigned long long write_reqs_submitted;
+	unsigned long long write_reqs_completed;
+	unsigned long long write_sectors;
+	unsigned long long write_total_ticks;
+	uint64_t io_errors;
+	uint64_t flags;
+};
```

Comment:
Can we not get this struct definition from tapdisk-metrics.h? I thought the plan was to add that to blktap-devel. Having two separate definitions of the same struct means they're likely to go out of sync.